# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../at76c503a/no-module-mv.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Instead of extra efford to just make this a no op, we patch it away ...

  - Rene Rebe <rene@exactcode.de>

--- at76c503a/Makefile.vanilla	2005-05-21 19:20:06.000000000 +0200
+++ at76c503a/Makefile	2005-05-21 19:20:16.000000000 +0200
@@ -91,7 +91,7 @@
 	for i in $(MODULES:%.o=%.ko); do \
 	  find $(DEL_MODULE_DIR) -name $$i -exec rm -f {} {}.orig \; ;done 
 
-install-modules: all mv_old
+install-modules:
 	mkdir -p $(MODULE_DIR)
 	install -m 644 -o 0 -g 0 $(MODULES:%.o=%.ko) $(MODULE_DIR)
 
@@ -128,7 +128,7 @@
 	for i in $(MODULES); do \
 	  find $(DEL_MODULE_DIR) -name $$i -exec rm -f {} {}.orig \; ;done 
 
-install-modules: all mv_old
+install-modules:
 	mkdir -p $(MODULE_DIR)
 	for f in $(MODULES); do install -m 644 -o 0 -g 0 $$f $(MODULE_DIR)/$$f; done
 
